src/sagas/sagas.js

```diff
@@ -1,15 +1,34 @@
-import { put, takeEvery, all } from 'redux-saga/effects';
-import { createQuestion } from '../actions';
+import { put, takeEvery, all, select, call } from 'redux-saga/effects';
 
-export function* createSagaQuestion(action) {
-  console.log('the fox and the lion');
-  yield put({ type: createQuestion.type, payload: action.payload });
+import { saveState, loadState } from '../util/localStorage';
+
+import { initialState } from '../reducer/RejectionReducer';
+
+export const getQuestions = state => state.questions;
+
+export function* persistState({ type }) {
+  if (!type.includes('LOAD') && !type.includes('INIT')) {
```

Comment:
whitelist this, don't do *, as it is more future proof, and you'll have to update this logic. Could grow bugs. 
If you do whitelist, you might have a problem however still no guard clause